### ISSUE TITLE
Problem faced when executing runtest.sh under wsl

### ISSUE THREAD

**Issue by xuanqi966** [nus-cs2103-AY2021S2/forum#18]
![testingScreenShot](https://user-images.githubusercontent.com/58899539/104983074-ff28b300-5a46-11eb-8520-04f558752949.png)
Hi, I'm facing this issue when executing runtest.sh under wsl. I have only modified input.txt and EXPECTED.txt and nothing else. Wonder if anyone is facing similar issue.

**Comment by tjtanjin** [comment]
Hello!

Not familiar with WSL but are there any solutions you have tried and what are you getting when you type java instead of javac on the command prompt? Hopefully the extra information will help spur potential answers though lots of results online seem to be pointing this as a path issue :3

**Comment by damithc** [comment]
General comment: Googling for `javac command not found wsl` might get you to some potential solutions

**Comment by xuanqi966** [comment]
I have resolved the issue, it's because I did not install packages such as jdk11 or dos2unix under my WSL but only on Windows. Some learning points for people who might also face such issues under WSL:

If the output says command not found, navigate to that specific line and run that command in your WSL to see what is the issue. Most of the time the system will tell you where to look for solutions.

Thank you @damithc @tjtanjin !
